### PR TITLE
Newline not added to .gitignore entry on add

### PR DESCRIPTION
**Pull request by pedroos**
Adding newline to printf instruction

What does this implement/fix? Explain your changes.
---------------------------------------------------

This fixes not adding new lines in automatic .gitignore inclusions by git-secret.

Does this close any currently open issues?
------------------------------------------

Issue #643.

### CODE REVIEW

**Comment by joshrabinowitz** [comment]
@pedroos this is great. 

One thing I'd like to add is a test that shows that .gitignore files look as expected with this patch. If you want to add that it'd be cool; the tests are pretty straightforward if you look at the current examples, and the bats-core docs (that's the library we use for testing) are pretty straightforward too.

Otherwise I'll try to write the tests after we manually test and probably merge this.

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 ### Bugfixes
 
+- Fix adding newlines to .gitignore entries
```
Comment (joshrabinowitz) [inline CHANGELOG.md:7]: This line should end with (#643)

**Comment by joshrabinowitz** [comment]
@sobolevn can you take a look at this PR and accept/reject?

**Comment by sobolevn** [comment]
@joshrabinowitz looks good!

> Otherwise I'll try to write the tests after we manually test and probably merge this.

Will you be able to do this?

**Comment by joshrabinowitz** [comment]
> @joshrabinowitz looks good!
> > Otherwise I'll try to write the tests after we manually test and probably merge this.
> Will you be able to do this?

@sobolevn: I've been busy with other projects, so it's not on my high-priority list. 
I was hoping you could take this PR over!

**Comment by sobolevn** [comment]
Rebased! I have added a new check to ensure that our `.gitignore` looks like the one we expect.

P.S. I hate escaping rules in `bash` 😢

**Comment by sobolevn** [comment]
@joshrabinowitz please, merge when you will have the time to look thriugh the changes.

**Comment by joshrabinowitz** [comment]
@sobolevn merged!

**Pull request merged**